packages/core/src/external/carequality/command/documents/send-ihe-gateway-results.ts

```diff
@@ -0,0 +1,128 @@
+import { Sequelize, QueryTypes } from "sequelize";
+import axios from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { RaceControl, checkIfRaceIsComplete, controlDuration } from "../../../../util/race-control";
+import { MetriportError } from "../../../../util/error/metriport-error";
+import { initSequelizeForLambda } from "../../../../util/sequelize";
+import { errorToString } from "../../../../util/error/shared";
+
+const REQUEST_ID_COLUMN = "request_id";
+
+const api = axios.create();
+
+dayjs.extend(duration);
+
+export const CONTROL_TIMEOUT = dayjs.duration({ minutes: 3 });
```

Comment:
these are important timeouts, why 3 minutes in specific?

Reply:
Good point i dont recall the reasoning for the exact number i beleive i saw this somewhere else in the logic but i can up it to like maybe 15 minutes?